shared/src/commonMain/kotlin/io.newm.shared/public/analytics/events/AppScreens.kt

```diff
@@ -5,28 +5,113 @@ import io.newm.shared.public.analytics.ScreenEvents
 
 object AppScreens {
```

Comment:
@martyu I added all the analytic events to this class in KMP so you can reuse the same screen names and button clicks. Hope this helps